api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java

```diff
@@ -29,12 +29,12 @@
 
 @Component
 @Setter(AccessLevel.PACKAGE)
+@SuppressWarnings("unchecked")
 public class FhirPersonDaoImpl extends BasePersonDao<Person> implements FhirPersonDao {
```

Comment:
So, to do this properly, we actually need to make `BasePersonDao` extend `BaseFhirDao`

Reply:
Are you trying to mean i extend BaseFhirDao instead of BasePersonDao

Reply:
No. I'm saying you need to change the `BasePersonDao` class so that it extends `BaseFhirDao`.